docs/en-US/component/upload.md

```diff
@@ -123,8 +131,9 @@ upload/manual
 
 ## Methods
 
-| Methods Name | Description                                                                             | Parameters                  |
-| ------------ | --------------------------------------------------------------------------------------- | --------------------------- |
-| clearFiles   | clear the uploaded file list (this method is not supported in the `before-upload` hook) | —                           |
-| abort        | cancel upload request                                                                   | （ file: fileList's item ） |
-| submit       | upload the file list manually                                                           | —                           |
+| Methods Name | Description                                                                     | Parameters                                                                    | Default             |
+| ------------ | ------------------------------------------------------------------------------- | ----------------------------------------------------------------------------- | ------------------- |
+| clearFiles   | clear the file list (this method is not supported in the `before-upload` hook). | UploadStatus[] (UploadStatus = 'ready' \| 'uploading' \| 'success' \| 'fail') | ['success', 'fail'] |
```

Comment:
The default value should be `['success', 'fail', 'ready', 'uploading']`  rel #4365

Reply:
ok